client/common/components/MainDashBoard.tsx

```diff
@@ -2,13 +2,13 @@ import React from 'react';
 import styles from '../../styles/MainDashBoard.module.css';
 import MileStoneCard from './MileStoneCard';
 
-const MainDashboard = () => {
+const MainDashboard = ({data}: any) => {
   return (
     <div className={styles.container}>
-      <MileStoneCard title="Milestone Title1" />
-      <MileStoneCard title="Title2" />
-      <MileStoneCard title="Title3" />
-      <MileStoneCard title="Title4" />
+      <MileStoneCard title={data[0].title} issue={data[1].tasks[0]} />
+      <MileStoneCard title={data[1].title} issue={data[0]} />
+      <MileStoneCard title={data[2].title} issue={data[0]} />
+      <MileStoneCard title={data[3].title} issue={data[0]} />
     </div>
```

Comment:
this would be in a loop and we would first check if there is a milestone in the data